apps/site/util/imageUtils.ts

```diff
@@ -0,0 +1,13 @@
+export const isSvgImage = (src: string) => {
+  let isSvg = false;
+
+  if (src.includes('.svg')) {
+    const image = new URL(src);
```

Comment:
I don't think you should do this, this will error for many images that might not be valid URLs per-se.